src/ui/NoNameFacet/NoNameFacetHeader/NoNameFacetHeader.ts

```diff
@@ -0,0 +1,76 @@
+import { $$ } from '../../../utils/Dom';
+import { l } from '../../../strings/Strings';
+import { SVGIcons } from '../../../utils/SVGIcons';
+import { SVGDom } from '../../../utils/SVGDom';
+import { INoNameFacetOptions } from '../NoNameFacetOptions';
+import { NoNameFacetHeaderClear } from './NoNameFacetHeaderClear';
+import { NoNameFacetHeaderCollapseToggle } from './NoNameFacetHeaderCollapseToggle';
+import { NoNameFacetHeaderOperatorToggle } from './NoNameFacetHeaderOperatorToggle';
+
+export interface INoNameFacetHeaderOptions {
+  rootFacetOptions: INoNameFacetOptions;
+}
+
+export class NoNameFacetHeader {
+  public element: HTMLElement;
+  private waitAnimationElement: HTMLElement;
+  private clearButton: NoNameFacetHeaderClear;
+  private operatorToggle: NoNameFacetHeaderOperatorToggle;
+  private collapseToggle: NoNameFacetHeaderCollapseToggle;
+
+  constructor(private options: INoNameFacetHeaderOptions) {
+    this.element = $$('div', { className: 'coveo-facet-header' }).el;
+    $$(this.element).append(this.createTitleSection());
+    $$(this.element).append(this.createSettingsSection());
+  }
+
+  private createTitleSection() {
+    const section = $$('div', { className: 'coveo-facet-header-title-section' });
+
+    section.append(this.createTitle());
+
+    this.waitAnimationElement = this.createWaitAnimation();
+    section.append(this.waitAnimationElement);
+    this.toggleLoading(false);
+
+    return section.el;
+  }
+
+  private createSettingsSection() {
+    const { useAnd, isCollapsed, enableCollapse, enableOperator } = this.options.rootFacetOptions;
+    const section = $$('div', { className: 'coveo-facet-header-settings-section' });
+
+    this.clearButton = new NoNameFacetHeaderClear();
+    section.append(this.clearButton.create());
+
+    this.operatorToggle = new NoNameFacetHeaderOperatorToggle({ useAnd });
```

Comment:
I would vote to have no option/functionnality to allow toggling operator between and / or.

We still allow to configure the operator on the facet component wen it gets initialized, but we remove the possibility for the end user to switch between the two.

Reply:
Im very happy to remove this